Screens/CreateProfile.js

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import { StyleSheet } from 'react-native';
+import { Headline, Paragraph, TextInput } from 'react-native-paper';
+import { RectangleButton } from '../Components';
+
+export default class CreateProfile extends React.Component {
+    
+    static navigationOptions = {
+      title:'Create a Profile',
+    };
+
+    render() {
+      return (
+        <View style={styles.wrapper}>
+        <View style={styles.container}>
+          <Headline style={styles.title}>
+          Welcome!
+          </Headline>    
+          <Paragraph>Since this is your first time, we just need some basic information from you.</Paragraph>
+          <TextInput
+          label='Full Name'
+          style={[styles.textInput, {marginTop:0, marginBottom:0}]} 
+          />        
+          <TextInput
+          style={[styles.textInput, {marginTop:0, marginBottom:0}]} 
```

Comment:
Weird that `textInput` sets a marginTop and marginBottom and you override it. Seems fishy.